.gitignore

```diff
@@ -6,11 +6,13 @@
 .packages
 .pub/
 build/
-ios/.generated/
 packages
 .flutter-plugins
 
 #Ignore IOS specific files
 ios/Podfile
 ios/Podfile.lock
 ios/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
+ios/Runner.xcodeproj/project.pbxproj
+ios/.generated/
+pubspec.lock
```

Comment:
I have read somewhere, that pubspec.lock should stay in version control... But I'm not quite sure about it...